Initilize commit

namonak opened the pull request:
---
name: Pull request
about: Create a pull request
title: 'first pull request'
labels: ''
assignees:  'zrma'

---

**Is your pull request related to an issue?**
A clear and write issue link like #[number]

**Describe what does this PR if not ralated to an issue.
A clear and concise description of what does this PR do

namonak commented:
@zrma 

I applied the prefix from the latest commit message.

namonak commented:
I added a prefix to all log message I committed.
8238c82197345f7524d2df1337f8740bbec78e82
6670eca49db8c93538d6b1a89d57fed3b372f70e

zrma commented:
> I added a prefix to all log message I committed.
> [8238c82](https://github.com/1day1coding/keepboard/commit/8238c82197345f7524d2df1337f8740bbec78e82)
> [6670eca](https://github.com/1day1coding/keepboard/commit/6670eca49db8c93538d6b1a89d57fed3b372f70e)

You missed your first commit to append prefix
9f40160cd0675e52dfbd236c0d84f9785cb796d5

zrma commented on server/go.mod:
```diff
@@ -0,0 +1,9 @@
+module server
+
+go 1.12
```
@namonak 

I recommend you to migrate go version 1.12 to 1.14

namonak replied:
> @namonak
> 
> I recommend you to migrate go version 1.12 to 1.14

@zrma
Following your advice, I upgraded the GO version to 1.14.
6418052

zrma commented on server/src/main.go:
```diff
@@ -0,0 +1,15 @@
+
+func main() {
+	e := echo.New()
+	e.GET("/", func(c echo.Context) error {
```
@namonak 

I recommend you to add some test code of this.
if you use goland, you can use jetbrains embeded http test client.

ref: #19 backend/cmd/server/main.http
ref: https://github.com/1day1coding/keepboard/blob/feature/5_axios/backend/cmd/server/main.http

namonak replied:
> recommend you to add some test code of this.
> if you use goland, you can use jetbrains embeded http test client.

I Added test code accoding to your comment.
1b4c7024699899909aafbcaa6ba311c37e263bf5

namonak commented:
> @namonak
> 
> > I added a prefix to all log message I committed.
> > [8238c82](https://github.com/1day1coding/keepboard/commit/8238c82197345f7524d2df1337f8740bbec78e82)
> > [6670eca](https://github.com/1day1coding/keepboard/commit/6670eca49db8c93538d6b1a89d57fed3b372f70e)
> 
> You missed your first commit to append prefix
> [9f40160](https://github.com/1day1coding/keepboard/commit/9f40160cd0675e52dfbd236c0d84f9785cb796d5)
> 
> and I recommend modify your first commit like this
> 
> > backend: initialize serverside sample code, etc, blablabla

Modified to fit the your recommended template!

namonak commented:
@zrma 
Following your advice, I upgraded the GO version to 1.14.
6418052ee13643ff1392be53b4575b11391861f5

namonak commented:
@zrma 
Thank you for your support and comment!

namonak commented:
I will try 'Rebase and Merge'
